scipy/stats/__init__.py

```diff
@@ -297,6 +297,7 @@
    brunnermunzel
    combine_pvalues
    jarque_bera
+   poisson_means_test
```

Comment:
This is where we should also be adding to `__all__`, not in `stats.py`. I'll submit a PR to import all of `_hypotests` so you can remove the changes in `stats.py`.